### PR TITLE
Feature optimization cssandcomponents2

### PR DESCRIPTION
**Pull request by andasan**

### CODE REVIEW

**Comment by taishikato** [comment]
How can I see the login page?
It keeps showing ↓ image.
I read code and seems that `loading` keeps being `true` when you are not logged in because `login` is called only when `user` is not null?
https://github.com/Final-Project-CICCC/shouter/blob/develop/src/App.js#L16-L20

Screen Shot 2563-03-18 at 15 13 48

**Comment by taishikato** [comment]
@andasan by the way, have you installed the prettier plugin on VSCode?
Please do it if not 👍

File: src/App.js [hunk 1]
```diff
@@ -1,29 +1,31 @@
...
-import {LoginPage} from './components/LoginPage/LoginPage';
+import { LoginPage } from './components/LoginPage/LoginPage';
 import Timeline from './components/Timeline';
 import Profile from './components/Profile';
-
 import PrivateRoute from './components/PrivateRoute';
 import PublicRoute from './components/PublicRoute';
 import firebase from './plugins/firebase';
-import {AuthContext} from './contexts/AuthContext';
+import { AuthContext } from './contexts/AuthContext';
```
Comment (taishikato) [inline src/App.js]: These spaces must be removed by prettier…

File: src/App.js [hunk 2]
```diff
@@ -1,29 +1,31 @@
-import React, {useEffect, useContext} from 'react';
-import {BrowserRouter, Switch} from 'react-router-dom';
+import React, { useEffect, useContext, useState } from 'react';
+import { BrowserRouter, Switch } from 'react-router-dom';
```
Comment (taishikato) [inline src/App.js]: These spaces must be removed by prettier…

Comment (andasan) [reply src/App.js]: i have prettier... but it keeps saving with the space for App.js 😔💁

Comment (taishikato) [reply src/App.js]: mm
Do you have a `.prettierrc` file on your root?
```json
{
  "printWidth": 120,
  "semi": true,
  "singleQuote": true,
  "trailingComma": "all",
  "bracketSpacing": false,
  "jsxBracketSameLine": true,
  "arrowParens": "avoid",
  "tabWidth": 2,
  "useTabs": false
}
```

Comment (taishikato) [reply src/App.js]: And have you installed ESlint for VSCode?

Comment (andasan) [reply src/App.js]: yes, i have the same rules for prettier on my .prettierrc

also, just installed ESlint, thanks ^^

**Review by taishikato — APPROVED** [review]

**Pull request merged**